spec/hutch/worker_spec.rb

```diff
@@ -45,9 +45,17 @@
     end
 
     it 'sets up a subscription' do
-      expect(queue).to receive(:subscribe).with(manual_ack: true)
+      expect(queue).to receive(:subscribe).with(consumer_tag: %r(^hutch\-.{36}$), manual_ack: true)
```

Comment:
So, now we know that the longest prefix that can be used is: 256 - 1 - 36 = 219 bytes.